src/main/java/org/jitsi/videobridge/IceUdpTransportManager.java

```diff
@@ -298,6 +298,37 @@ else if (strategy != null)
         }
     }
 
+    /**
+     * Stops the static <tt>Harvester</tt> instances used by all
+     * <tt>IceUdpTransportManager</tt> instances, that is
+     * {@link #tcpHarvester} and {@link #singlePortHarvesters}.
+     *
+     * @param cfg the {@link ConfigurationService} which provides values to
+     * configurable properties of the behavior/logic of the method
+     * implementation
+     */
+    public static void closeStaticConfiguration(ConfigurationService cfg)
+    {
+        synchronized (IceUdpTransportManager.class)
+        {
+            if (!staticConfigurationInitialized)
+            {
+                return;
+            }
+            staticConfigurationInitialized = false;
+
+            if (singlePortHarvesters != null)
+            {
+                singlePortHarvesters.forEach(AbstractUdpListener::close);
+            }
+
+            if (tcpHarvester != null)
+            {
+                tcpHarvester.close();
```

Comment:
Can you also set `tcpHarvester` and `singlePortHarvesters` to null?

I wonder if we should also reset the `healthy` flag.

Reply:
I think we should reset the `healthy` flag. Its purpose is to trigger (be false) only when the single port harvester is configured but failed to bind.

LGTO apart from this. The failed tests are likely unrelated.